test/images/echoserver/Dockerfile_windows

```diff
@@ -19,21 +19,24 @@ ARG REGISTRY
 FROM --platform=linux/amd64 alpine:3.6 as prep
 
 ADD https://openresty.org/download/openresty-1.13.6.2-win64.zip /openresty-win64.zip
+ADD http://wiki.overbyte.eu/arch/openssl-1.1.1i-win64.zip /openssl.zip
```

Comment:
thanks looks soooo shady!! :(

Reply:
I got this URL from here: https://wiki.openssl.org/index.php/Binaries

So, it should be fine, IMO.